app/uk/gov/hmrc/customs/emailfrontend/model/Ineligible.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.customs.emailfrontend.model
+
+import play.api.libs.json.{Reads, Writes}
+import play.api.mvc.{PathBindable, QueryStringBindable}
+
+object Ineligible extends Enumeration {
+  val NoEnrolment, IsAgent,NotAdmin = Value
+
+  implicit val reads: Reads[Ineligible.Value] = Reads.enumNameReads(Ineligible)
+  implicit val writes: Writes[Ineligible.Value] = Writes.enumNameWrites
+
+  implicit lazy val pathBindable: PathBindable[Ineligible.Value] = new PathBindable[Ineligible.Value] {
+
+    override def bind(key: String, value: String): Either[String, Ineligible.Value] =
```

Comment:
don't we need to bind this with the key being passed, as `key` parameter not being referenced in the implementation unless it's not needed there (not sure)